docs/UserGuide.md

```diff
@@ -73,19 +73,20 @@ Shows a message explaning how to access the help page.
 Format: `help`
 
 
-### Adding a person: `add`
+### Create Patient Record: `create -p`
 
-Adds a person to the address book.
+Adds a Patient to the record system.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `create -p f/FIRST_NAME n/GIVEN_NAME m/[MEDICAL_HISTORY]`
```

Comment:
Hi @rish-16, as communicated, let's keep to ```add``` being the command word: so ```add -p``` instead of ```create -p```

Thank you.